hack/install_kustomize.sh

```diff
@@ -48,6 +48,10 @@ fi
 function readlink_f {
   TARGET_FILE=$1
 
+  # Unset CDPATH to restore default cd behavior. An exported CDPATH can
+  # cause cd to output the current directory to STDOUT.
+  unset CDPATH
+
```

Comment:
I would just do this at the top of the script, to defend against future additions of `cd` commands elsewhere.